include/ocpp/v201/smart_charging.hpp

```diff
@@ -81,7 +81,12 @@ class SmartChargingHandler {
     ///
     /// \brief Adds a given \p profile and associated \p evse_id to our stored list of profiles
     ///
-    void add_profile(int32_t evse_id, ChargingProfile& profile);
+    ProfileValidationResultEnum add_profile(int32_t evse_id, ChargingProfile& profile);
+
+    ///
+    /// \brief Retrieves existing profiles on system.
+    ///
+    ChargingProfile get_profiles();
```

Comment:
Where is the implementation for `get_profiles ()`?

Reply:
Ok, so now that I am actually building my code before running tests, I was able to actually test-drive this code AND actually have it exist.